modules/gitops-release-notes-1-14-2.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes-1-14.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-14-2_{context}"]
+= Release notes for {gitops-title} 1.14.2
+
+{gitops-title} 1.14.2 is now available on {OCP} 4.13, 4.14, 4.15, 4.16, and 4.17.
+
+[id="errata-updates-1-14.2_{context}"]
+== Errata updates
+
+[id="RHBA-XXXX:NNNN-gitops-1-14-2-security-update-advisory_{context}"]
+=== RHBA-XXXX:NNNN - {gitops-title} 1.14.2 security update advisory
+
+Issued: 2024-11-11
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHBA-XXXX:NNNN[RHBA-XXXX:NNNN]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="fixed-issues-1-14-2_{context}"]
+== Fixed issues
```

Comment:
@Dhruv-Soni11 , we also need to add https://issues.redhat.com/browse/GITOPS-3465 to 1.14.2 release notes.

Reply:
Fixed it now @ishitasequeira. Please take a look after some time when the build is ready.